routes/user/user.js

```diff
@@ -52,25 +59,30 @@ passport.use(new TokenStrategy(
  * Setup the local strategy required for login and establish checks
  * for credentials provided during login.
  */
-passport.use(new LocalStrategy(
-    function(username, password, done) {
+passport.use(new LocalStrategy({passReqToCallback:true},
+    function(req, username, password, done) {
         User.findOne({ us_username: username }, 'us_username ' +
             'us_user_first_name us_user_last_name us_email_address us_contact' +
             'us_user_role us_state us_password' ,
             function (err, user) {
                 console.log(err, user);
                 if (err) {
+                    logging.accessLogger(null,req.url,"user_activity", "A database error occurred while authenticating.",false);
                     return done(err);
                 }
                 if (!user) {
+                    logging.accessLogger(null,req.url,"user_activity", "Incorrect user credentials supplied.",false);
                     return done(null, false, {message: 'Incorrect credentials.'});
                 }
                 if (user.us_password != password) {
+                    logging.accessLogger(null,req.url,"user_activity", "Incorrect user credentials supplied.",false);
                     return done(null, false, {message: 'Incorrect credentials.'});
                 }
                 if (user.us_state != 'active') {
+                    logging.accessLogger(user._id, req.url,"user_activity", "User account requires activation",false);
                     return done(null, false, {message: 'User Activation Required.'});
                 }
+                else logging.accessLogger(user._id, req.url,"user_activity", "User Login Successful",true);
```

Comment:
@nick22891 for all the possible values for `lo_log_level` create a set of constants that exists within the logging file that you can refer to.